djangoProject/.gitignore

```diff
@@ -0,0 +1,2 @@
+# Environment variables
+.env
```

Comment:
please update the file .gitignore
you can use this sample:
https://github.com/django-json-api/django-rest-framework-json-api/blob/main/.gitignore